common/src/main/java/com/wynntils/wynn/model/questbook/QuestBookModel.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.wynn.model.questbook;
+
+import com.wynntils.core.WynntilsMod;
+import com.wynntils.core.managers.Model;
+import com.wynntils.wynn.event.QuestBookReloadedEvent;
+import com.wynntils.wynn.model.container.ContainerContent;
+import com.wynntils.wynn.model.container.ScriptedContainerQuery;
+import com.wynntils.wynn.utils.InventoryUtils;
+import java.util.ArrayList;
+import java.util.List;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.item.Items;
+
+public class QuestBookModel extends Model {
+    private static final int NEXT_PAGE_SLOT = 8;
+
+    private static List<QuestInfo> quests = List.of();
+    private static List<QuestInfo> newQuests;
+
+    public static List<QuestInfo> getQuests() {
+        return quests;
+    }
```

Comment:
Can we move this to the other getters?

Reply:
I reordered it, but I'm not sure it got better. This one is a public getter, one of the two public methods on the manager. The other to methods named "get*" are more like "calculateThisString" than reading a field like a normal getter.